x-pack/test/functional/apps/endpoint/alert_list.ts

```diff
@@ -22,6 +22,10 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
     it('loads the Alert List Page', async () => {
       await testSubjects.existOrFail('alertListPage');
     });
+    it('verifies the Alert List Page title', async () => {
```

Comment:
The way this test description is written will cause the test execution to read like this:
```
Endpoint Alert List verifies the Alert List Page title
```
I suggest updating the description such that the execution read better. Maybe something like:
```
Endpoint Alert List contains the Alerts title
```

Reply:
That's better than what I have, will update shortly. Thanks P.

Reply:
i modified, lets see how we like the latest when folks have time to review